### PR TITLE
chore(localdebug): update the output messages of local tunnel task

### PR DESCRIPTION
**Pull request by XiaofuHuang**
1. Start ngrok success
Output: 
image
Terminal:
image

 
2. ngrok install failed
 Output: 
image
Terminal:
image

3. ngrok start failed
 Output: 
image
Terminal:
image

 
4. Start dev tunnel success
 Output: 
image
Terminal:
image

 
5. Start dev tunnel failed
 Output: 
image
 Terminal:
image

### CODE REVIEW

File: packages/vscode-extension/package.nls.json [hunk 1]
```diff
@@ -267,6 +267,20 @@
     "teamstoolkit.localDebug.tunnelServiceNotStartedError": "The tunnel service hasn't been started. Please wait for a moment or restart the local tunnel task.",
     "teamstoolkit.localDebug.tunnelEndpointNotFoundError": "Failed to find the tunnel endpoint. Teams toolkit try to get the first HTTPS URL from %s and failed.",
     "teamstoolkit.localDebug.tunnelEnvError": "Failed to save environment variables. %s",
+    "teamstoolkit.localDebug.startTunnelError": "Start local tunnel task failed. %s",
```
Comment (sffamily) [inline packages/vscode-extension/package.nls.json:270]: Task to start local tunneling failed.

File: packages/vscode-extension/package.nls.json [hunk 2]
```diff
@@ -267,6 +267,20 @@
...
+    "teamstoolkit.localDebug.output.tunnel.title": "Running 'Start local tunnel' Visual Studio Code task.",
```
Comment (sffamily) [inline packages/vscode-extension/package.nls.json:271]: Running Visual Studio Code task: 'Start local tunnel'.

File: packages/vscode-extension/package.nls.json [hunk 3]
```diff
@@ -267,6 +267,20 @@
...
+    "teamstoolkit.localDebug.output.tunnel.checkNumber": "Teams Toolkit is starting the local tunnel service to forward public URL to local port and inspect traffic. Open terminal window to see the running details.",
+    "teamstoolkit.localDebug.output.summary": "Summary:",
+    "teamstoolkit.localDebug.output.tunnel.learnMore": "Visit %s to learn more about 'Start local tunnel' task.",
+    "teamstoolkit.localDebug.output.tunnel.successSummary": "Forwarding URL %s to %s.",
+    "teamstoolkit.localDebug.output.tunnel.successSummaryWithEnv": "Forwarding URL %s to %s. Saved [%s] to %s.",
+    "teamstoolkit.localDebug.output.tunnel.duration": "Started local tunnel service in %s seconds.",
```
Comment (sffamily) [inline packages/vscode-extension/package.nls.json:277]: tunnel -> tunneling

**Review by a1exwang — APPROVED** [review]

File: packages/vscode-extension/package.nls.json (hunk 1 above)
Comment (XiaofuHuang) [reply packages/vscode-extension/package.nls.json]: Updated.

File: packages/vscode-extension/package.nls.json (hunk 2 above)
Comment (XiaofuHuang) [reply packages/vscode-extension/package.nls.json]: Updated.

File: packages/vscode-extension/package.nls.json (hunk 3 above)
Comment (XiaofuHuang) [reply packages/vscode-extension/package.nls.json]: Updated.

**Review by a1exwang — DISMISSED** [review]

File: packages/vscode-extension/package.nls.json [hunk 4]
```diff
@@ -267,6 +267,20 @@
     "teamstoolkit.localDebug.tunnelServiceNotStartedError": "The tunnel service hasn't been started. Please wait for a moment or restart the local tunnel task.",
```
Comment (sffamily) [inline packages/vscode-extension/package.nls.json:267]: Can remove the ‘please’

File: packages/vscode-extension/package.nls.json [hunk 5]
```diff
@@ -267,6 +267,20 @@
...
     "teamstoolkit.localDebug.tunnelEndpointNotFoundError": "Failed to find the tunnel endpoint. Teams toolkit try to get the first HTTPS URL from %s and failed.",
```
Comment (sffamily) [inline packages/vscode-extension/package.nls.json:268]: try -> tried

File: packages/vscode-extension/package.nls.json [hunk 6]
```diff
@@ -267,6 +267,20 @@
...
     "teamstoolkit.localDebug.tunnelEnvError": "Failed to save environment variables. %s",
+    "teamstoolkit.localDebug.startTunnelError": "Task to start local tunneling service failed. %s",
+    "teamstoolkit.localDebug.output.tunnel.title": "Running Visual Studio Code task: 'Start local tunnel'",
+    "teamstoolkit.localDebug.output.tunnel.checkNumber": "Teams Toolkit is starting the local tunneling service to forward public URL to local port and inspect traffic. Open terminal window to see the running details.",
```
Comment (sffamily) [inline packages/vscode-extension/package.nls.json:272]: Can remove ‘running’

File: packages/vscode-extension/package.nls.json [hunk 7]
```diff
@@ -267,6 +267,20 @@
...
+    "teamstoolkit.localDebug.output.tunnel.duration": "Started local tunneling service in %s seconds.",
+    "teamstoolkit.localDebug.output.tunnel.startDevTunnel": "Starting dev tunnel service",
+    "teamstoolkit.localDebug.output.tunnel.startNgrokMessage": "Starting ngrok service",
+    "teamstoolkit.localDebug.output.tunnel.checkNgrokMessage": "Checking and installing ngrok",
+    "teamstoolkit.localDebug.output.tunnel.installSuccessMessage": "ngrok is installed at %s.",
+    "teamstoolkit.localDebug.output.tunnel.skipInstallMessage": "Skip checking and installing ngrok as user has specified ngrok path (%s).",
+    "teamstoolkit.localDebug.output.tunnel.createDevTunnelMessage": "Dev tunnel tag: %s.",
     "teamstoolkit.localDebug.launchTeamsWebClientError": "Failed to launch Teams web client.",
     "teamstoolkit.localDebug.launchTeamsWebClientStoppedError": "Task to launch Teams web client stopped with exit code '%s'.",
     "teamstoolkit.migrateTeamsManifest.progressTitle": "Upgrade Teams Manifest to Extend in Outlook and the Microsoft 365 app",
```
Comment (sffamily) [inline packages/vscode-extension/package.nls.json:286]: Lower case for Extend’s E

File: packages/vscode-extension/package.nls.json [hunk 8]
```diff
@@ -267,6 +267,20 @@
...
     "teamstoolkit.localDebug.tunnelEnvError": "Failed to save environment variables. %s",
+    "teamstoolkit.localDebug.startTunnelError": "Task to start local tunneling service failed. %s",
+    "teamstoolkit.localDebug.output.tunnel.title": "Running Visual Studio Code task: 'Start local tunnel'",
+    "teamstoolkit.localDebug.output.tunnel.checkNumber": "Teams Toolkit is starting the local tunneling service to forward public URL to local port and inspect traffic. Open terminal window to see the running details.",
+    "teamstoolkit.localDebug.output.summary": "Summary:",
+    "teamstoolkit.localDebug.output.tunnel.learnMore": "Visit %s to learn more about 'Start local tunnel' task.",
+    "teamstoolkit.localDebug.output.tunnel.successSummary": "Forwarding URL %s to %s.",
+    "teamstoolkit.localDebug.output.tunnel.successSummaryWithEnv": "Forwarding URL %s to %s. Saved [%s] to %s.",
+    "teamstoolkit.localDebug.output.tunnel.duration": "Started local tunneling service in %s seconds.",
+    "teamstoolkit.localDebug.output.tunnel.startDevTunnel": "Starting dev tunnel service",
```
Comment (sffamily) [inline packages/vscode-extension/package.nls.json:278]: tunneling

File: packages/vscode-extension/package.nls.json (hunk 4 above)
Comment (sffamily) [reply packages/vscode-extension/package.nls.json:267]: and tunnel -> tunneling

File: packages/vscode-extension/package.nls.json (hunk 8 above)
Comment (XiaofuHuang) [reply packages/vscode-extension/package.nls.json:278]: "Dev tunnel" is the current brand name. Should we consider changing it to "tunneling"?

**Review by sffamily — APPROVED** [review]

**Review by a1exwang — APPROVED** [review]

File: packages/vscode-extension/package.nls.json [hunk 9]
```diff
@@ -258,18 +258,32 @@
...
     "teamstoolkit.localDebug.ngrokProcessError": "Failed to start ngrok. %s",
     "teamstoolkit.localDebug.ngrokNotFoundError": "Ngrok is not installed by TeamsFx. See teamsfx-debug-tasks#debug-check-prerequisites for how to install the ngrok.",
     "teamstoolkit.localDebug.ngrokInstallationError": "Failed to install Ngrok. %s",
-    "teamstoolkit.localDebug.tunnelServiceNotStartedError": "The tunnel service hasn't been started. Please wait for a moment or restart the local tunnel task.",
-    "teamstoolkit.localDebug.tunnelEndpointNotFoundError": "Failed to find the tunnel endpoint. Teams toolkit try to get the first HTTPS URL from %s and failed.",
+    "teamstoolkit.localDebug.tunnelServiceNotStartedError": "The tunneling service hasn't been started. Wait for a moment or restart the local tunneling task.",
+    "teamstoolkit.localDebug.tunnelEndpointNotFoundError": "Failed to find the tunnel endpoint. Teams toolkit tried to get the first HTTPS URL from %s and failed.",
     "teamstoolkit.localDebug.tunnelEnvError": "Failed to save environment variables. %s",
+    "teamstoolkit.localDebug.startTunnelError": "Task to start local tunneling service failed. %s",
```
Comment (HuihuiWu-Microsoft) [inline packages/vscode-extension/package.nls.json:270]: Separate the two sentences if '%s' has no relationship with this string. Or it'll be confusing to localization team.

Comment (XiaofuHuang) [reply packages/vscode-extension/package.nls.json:270]: Thx, I will create another PR to change all the `%s` in localdebug

**Review by HuihuiWu-Microsoft — APPROVED** [review]
Please help update in another PR

**Pull request merged**